src/containerapp/azext_containerapp/_help.py

```diff
@@ -1684,3 +1684,132 @@
           az containerapp sessionpool list -g MyResourceGroup
 """
 
+# SessionPool Commands
+helps['containerapp sessionpool'] = """
+    type: group
+    short-summary: Commands to manage session pools.
+"""
+
+helps['containerapp sessionpool create'] = """
+    type: command
+    short-summary: Create or update a Session pool.
```

Comment:
In CLI, create command is only used for create operation
```suggestion
    short-summary: Create a Session pool.
```

Reply:
please check the similar messages as well.

Reply:
Hi @yanzhudd 

This command support update too.